web/EXAMPLES.md

```diff
@@ -130,19 +135,35 @@ The second rule is limited to blocking all matching requests except for those wh
 | example.com | test-tracker.net/ddm/ | image  |  false | matches rule and matches rule exception |
 | example.com | test-tracker.net/adimage.png | image  |  true | no matching rule, default is set to 'block' |
 
-### Rule with `exceptions`
-
-We have `default: ignore`, so we only block requests that match a rule. The first rule doesn't have any `exceptions`, so we block any matching request. The second rule has a blocking exception. We will not block matching `image` requests that come from `test-site-2.com` or its subdomains.
+### Rule with `exceptions` and `options`
 
+We have `default: ignore`, so we only block requests that match a rule. 
```

Comment:
why in the example below `default:ignore` is at the bottom? Makes it kinda hard to figure out on a first look what you are talking about. In the blocklsit action is above the rules. It's a nit but can we move it?